### ISSUE TITLE
Fake punch contact

### ISSUE THREAD

**Issue by shayanshokry** [efroemling/ballistica#615]
### Description
this bug exist in a long time (maybe even since the release who knows?)and nobody mentioned it or maybe i dont see anyone talking about it, its a about a time that sometimes your punch reaches the opponent and have some little spark effect to show the punch landed BUT the opponent takes 0 damage, its just a spark effect and nothing more, maybe the punch was so weak that it do 0 damage but still the spark effect exist, idk if its a punch problem or not, such a bug its not necessary or a big problem, its just exist and make some players confused thats all
also i don't installed any mods or plugins

### Steps to reproduce
its just happens randomly depends on the player's play style

### Expected behavior
maybe the spark effect shouldn't happen while this happens, i dont exactly know if the problem is with landing a punch or just the problem with spark effect timing.

### Machine
windows 11
android
there is no difference between the platform, its always happens randomly no matter what

### Extra
i think there is a lot of people that experienced this, if this happened to anyone please mention in there

**Comment by ravishyam30** [comment]
I also face this problem. When I punch no damage ever and loose every time. Please fix this so that I can become the G.O.A.T .

**Comment by SoK05** [comment]
Yeah this is a BombSquad quirk, but I agree it's quite annoying. Especially if you lose the fight because of the punch randomly not being accepted.

**Comment by MatiasMGz** [comment]
This should be fixed as soon as possible. Its annoying.
